webapp/channels/src/components/admin_console/team_channel_settings/group/group_list.tsx

```diff
@@ -47,7 +47,7 @@ type Props = {
     emptyListTextId: string;
     emptyListTextDefaultMessage: string;
     actions: {
-        getData: () => Promise<Group[]>;
+        getData: () => Promise<any>;
     };
     removeGroup: (gid: string) => void;
     setNewGroupRole: (gid: string) => void;
```

Comment:
kept most of the original props as `group_list_test` starts complaining if I use `type Props = OwbnProps & PropsFromRedux` instead. Not sure if we want to change `group_list_test` to accommodate using `type Props = OwbnProps & PropsFromRedux`.

Reply:
I would fix type issue in `group_list_test` instead